package.json

```diff
@@ -45,6 +45,7 @@
     "*.js": [
       "prettier --write",
       "eslint",
+      "jest --silent --coverage",
```

Comment:
Won't this run for each JS file that's changed? Should we instead configure Husky to run a single test command and *then* lint-staged for the precommit hook.

Reply:
@camacho Good catch. Have fixed this with the latest push which will only run hooks for staged files only.